docs/browser/events.md

```diff
@@ -0,0 +1,166 @@
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page - Initial page load within the browser and will generally also precede a PageNavigationTiming event.|
+| `1` | virtual_page - This is for Single Page Applications (SPA) where the framework provides the ability to perform client side only page "navigation", the exact definition of what a virtual page change is determined by the SPA and the framework it is using.|
+
+## PageNavigationTiming
+
+The event name MUST be `browser.page_navigation_timing`.
+
+This event describes the timing metrics of a page navigation as provided by
+[PerformanceNavigationTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming) Performance API.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| name | string | URL of the page || Recommended |
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`session.id`](../attributes-registry/session.md) | string | A unique id to identify a session. | `00112233-4455-6677-8899-aabbccddeeff` | `Opt-In` |
+| `browser.page.instance_id` | string | A unique id that identifies the instance of the page where the event is emitted. | `e611e02f-c9d2-47b3-a678-df1465e0f401` | `Opt-In` |
+
+## ResourceTiming
+
+The event name MUST be `browser.resource_timing`.
+
+This event describes the timing metrics as provided by [PerformanceResourceTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming)
+Performance API.  These metrics are related to fetching a resource, such as
+XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+|name | string | URL of the requested resource || Recommended |
+|fetchStart | long | || Recommended |
+|domainLookupStart | long | || Recommended |
+|domainLookupEnd | long | || Recommended |
+|connectStart | long | || Recommended |
+|secureConnectionStart | long | || Recommended |
+|connectEnd | long | || Recommended |
+|requestStart | long | || Recommended |
+|responseStart | long | || Recommended |
+|responseEnd | long | || Recommended |
```

Comment:
**Suggestion:** There are a few more resource timing fields that would be awesome to have/recommend ([spec](https://www.w3.org/TR/resource-timing/#webidl-1311849615)):

```
    readonly attribute [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long)  [transferSize](https://www.w3.org/TR/resource-timing/#dom-performanceresourcetiming-transfersize);
    readonly attribute [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long)  [encodedBodySize](https://www.w3.org/TR/resource-timing/#dom-performanceresourcetiming-encodedbodysize);
    readonly attribute [unsigned long long](https://webidl.spec.whatwg.org/#idl-unsigned-long-long)  [decodedBodySize](https://www.w3.org/TR/resource-timing/#dom-performanceresourcetiming-decodedbodysize);
    readonly attribute [unsigned short](https://webidl.spec.whatwg.org/#idl-unsigned-short) [responseStatus](https://www.w3.org/TR/resource-timing/#dom-performanceresourcetiming-responsestatus);
```

In reality though, most of the resource timings would be helpful, e.g., also `contentType`.

Reply:
I can add these, but a quick question to @MSNev, @martinkuba  and @open-telemetry/semconv-http-approvers -

These parameters are not really timing parameters, but http parameters. Are they better represented using `http.*` attributes on the `xhr`/`fetch`/`resourceFetch` spans?

Reply:
Added these fields.